### PR TITLE
added 9.3 installation instructions

### PR DESCRIPTION
**Pull request by j0eybrinkman**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/guides/web/php.md [hunk 1]
```diff
@@ -118,24 +192,23 @@ To serve PHP pages in CGI mode, you must install the apache server, configure it
 
 * Installation:
 
-```bash
-sudo dnf install httpd
-```
+	```bash
```
Comment (neelchauhan) [inline docs/guides/web/php.md:195]: You have many examples of unnecessary whitespace throughout this PR. This shouldn't be here.

**Review by neelchauhan — CHANGES_REQUESTED** [review]

File: docs/guides/web/php.md (hunk 1 above)
Comment (sspencerwire) [reply docs/guides/web/php.md:195]: > You have many examples of unnecessary whitespace throughout this PR. This shouldn't be here.

Actually, @neelchauhan, he is trying to implement this: https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/#tabbed-content-within-a-document

Which does require the four spaces he is showing. There's an issue with the tabs though, that I've outlined in another comment.

**Comment by j0eybrinkman** [comment]
> @j0eybrinkman This is good work! There's a slight problem though. I'd recommend editing this and keeping ALL of the 8.9 and 9.3 instructions together, rather than separating them throughout the document. If you want to do it the way you've got it written now, then the alternative change would be to change the tabs to describe the section, for example:
> 
> ```
> === "8.9 PHP module list" 
> ```
> 
> and
> 
> ```
> === "9.3 PHP module list"
> ```
> 
> But overall I think it is far better to only have the two tabs in the document (8.9 and 9.3) and have ALL of the instructions for each version contained inside
> 
> You can leave generic instructions (those that work for either version) out of the tabbed content.
> 
> Have you looked to see how this appears in the web interface? (In other words, have you successfully installed a local documentation implementation and seen what happens to this document as it is formatted? If not, then I would definitely suggest bundling all of the instructions for each version under one tab.

Yes I used the venv method to render the web page before submitting PR.

**Comment by j0eybrinkman** [comment]
Hi @sspencerwire I updated the tabs to be more descriptive. I do prefer the individual tabs for each section of the document rather than grouping them all together. Each set of commands is available under its respective section in the table of contents that way which I like.

**Comment by sspencerwire** [comment]
> Hi @sspencerwire I updated the tabs to be more descriptive. I do prefer the individual tabs for each section of the document rather than grouping them all together. Each set of commands is available under its respective section in the table of contents that way which I like.

Ok. Thanks @j0eybrinkman I'll approve the changes and merge them and take a look in the web env. :-)

**Review by sspencerwire — APPROVED** [review]
